package-lock.json

```diff
@@ -1087,6 +1086,746 @@
         "npm": "7.x || 8.x || 9.x"
       }
     },
+    "core/sandbox/node_modules/@dotcom-tool-kit/backend-heroku-app": {
```

Comment:
**issue**: why has the sandbox returned 😨

Reply:
ohhhhhhh. the sandbox is still gitignored, but i still have a `package.json` in that folder which is being picked up in the install. that's annoying. i can't think of a way to skip that without removing it from the workspace which kind of defeats the point of having it gitignored in the monorepo? we ignore these changes though

Reply:
could we exclude it in the `workspaces` value in the root `package.json`?

Reply:
i _think_ that would prevent it from picking up other workspace dependencies. we could use file version specifiers for those though?